packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -4485,6 +4485,259 @@ export class WalletService {
     });
   }
 
+  changellyGetKeys(req) {
+    if (!config.changelly) throw new Error('Changelly missing credentials');
```

Comment:
maybe better: "ClientError: Service not configured."
and log.warn: "Changelly missing credentials"

Reply:
Done